css/style.css

```diff
@@ -194,10 +378,90 @@ p.content-description {
 .popular-content-item .item-img {
     align-items: flex-end;
 }
-.services-button-list li {    
+.services {
+    position: relative;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 205px;
+    max-width: 1440px;
+    height: auto;
+}
+.services-container {
     display: flex;
+    justify-content: flex-start;
     align-items: center;
+    width: 1160px;
+    margin-left: auto;
+    margin-right: auto;
+    height: auto;
+}
+.services::before {
+    content: "";
+    display: block;
+    position: absolute;
+    width: 1440px;
+    height: 288px;
+    top: 70px;
+    background-color: #e5e5e5;
+    z-index: -1;
 }
+.services-button {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+    min-height: 180px;
+    min-width: 280px;
+    font-size: 18px;
```

Comment:
it's better to use font-family, and all set of font props in the elements with text content, like links or paragraphs of buttons

Reply:
OK, got it